backend/trafficrules/src/main/java/com/coffeejug/trafficrules/model/User.java

```diff
@@ -1,18 +1,33 @@
 package com.coffeejug.trafficrules.model;
 
+import java.time.LocalDateTime;
 import java.util.Objects;
 import java.util.UUID;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import org.hibernate.annotations.GenericGenerator;
 
 @Entity
 @Table(name = "users")
 public class User {
 
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(columnDefinition = "BINARY(16)")
+    private UUID uuid;
+    @Column(columnDefinition = "INT NOT NULL DEFAULT '0'")
+    private int levelsCompleted;
+    private String name;
+
+    private LocalDateTime registered;
```

Comment:
Can't we use `@CreatedAt` annotation here?

Reply:
I'm not sure about that